api/app/Actions/EventType/GetEventTypeByIdAction.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\EventType;
+
+use App\Actions\GetByIdRequest;
+use App\Repositories\EventType\EventTypeRepository;
+
+final class GetEventTypeByIdAction
+{
+    private $repository;
```

Comment:
Add type. Also, it would be good to rename it to $eventTypeRepository, as the other repositories can be added here in the future. Appliable to all actions.